packages/components/src/focal-point-picker/index.native.js

```diff
@@ -0,0 +1,264 @@
+	const focalPointStyles = [
+		styles.focalPoint,
+		{
+			height: FOCAL_POINT_SIZE,
+			marginLeft: -( FOCAL_POINT_SIZE / 2 ),
+			marginTop: -( FOCAL_POINT_SIZE / 2 ),
+			width: FOCAL_POINT_SIZE,
+		},
+	];
+
+	return (
+		<View style={ styles.container }>
+			<Tooltip visible={ tooltipVisible }>
+				<Tooltip.Overlay onPress={ () => setTooltipVisible( false ) }>
+					<View style={ [ styles.media, backgroundColor ] }>
+						<View
+							{ ...panResponder.panHandlers }
+							onLayout={ ( event ) => {
+								const {
+									height,
+									width,
+								} = event.nativeEvent.layout;
+
+								if (
+									width !== 0 &&
+									height !== 0 &&
+									( containerSize?.width !== width ||
+										containerSize?.height !== height )
+								) {
+									setContainerSize( { width, height } );
+								}
+							} }
+							style={ styles.mediaContainer }
+						>
+							{ ! isVideo && (
+								<Image
+									editButton={ false }
+									isSelected={ ! displayPlaceholder }
+									height="100%"
+									url={ url }
+									style={ imagePreviewStyles }
+									onImageDataLoad={ () => {
+										setDisplayPlaceholder( false );
+									} }
+								/>
+							) }
+
+							{ isVideo && (
+								<Video
+									muted
+									paused
+									disableFocus
+									onLoad={ ( event ) => {
+										const {
+											height,
+											width,
+										} = event.naturalSize;
+										setVideoNaturalSize( {
+											height,
+											width,
+										} );
+										setDisplayPlaceholder( false );
+									} }
+									resizeMode={ 'cover' }
+									source={ { uri: url } }
+									style={ videoPreviewStyles }
+								/>
+							) }
+							{ ! displayPlaceholder && (
+								<Animated.View
+									pointerEvents="none"
+									style={ focalPointGroupStyles }
+								>
+									<Tooltip.Label
+										text={ __(
+											'Drag to adjust focal point'
+										) }
+										yOffset={ -( FOCAL_POINT_SIZE / 2 ) }
+									/>
+									<FocalPoint
+										height={ styles.focalPoint.height }
+										style={ focalPointStyles }
+										width={ styles.focalPoint.width }
+									/>
+								</Animated.View>
+							) }
+						</View>
+					</View>
+					<RangeControl
+						inputSuffix="%"
+						key={ `xAxis-${ sliderKey }` }
+						label={ __( 'X-Axis Position' ) }
+						max={ MAX_POSITION_VALUE }
+						min={ MIN_POSITION_VALUE }
+						onChange={ ( x ) => setPosition( { x: x / 100 } ) }
+						value={ Math.round( focalPoint.x * 100 ) }
+					/>
+					<RangeControl
+						inputSuffix="%"
+						key={ `yAxis-${ sliderKey }` }
+						label={ __( 'Y-Axis Position' ) }
+						max={ MAX_POSITION_VALUE }
+						min={ MIN_POSITION_VALUE }
+						onChange={ ( y ) => setPosition( { y: y / 100 } ) }
+						value={ Math.round( focalPoint.y * 100 ) }
+					/>
+				</Tooltip.Overlay>
+			</Tooltip>
+		</View>
+	);
+}
+
+export default withPreferredColorScheme( FocalPointPicker );
```

Comment:
What do you think if we use the hook `usePreferredColorSchemeStyle` so we don't add a HOC?